src/app/index/index.component.css

```diff
@@ -4,5 +4,23 @@
 
 #search-bar{
   position: absolute;
-  left: 300px;
+}
+
+#set-susper-default{
+  display:none;
+}
+
+@media screen and (max-width: 480px) {
```

Comment:
Ok the third commit is not supposed to be in this PR. How can I discard it from this PR? Help needed.